gen/calc.py

```diff
@@ -1077,6 +1148,10 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
                 'service': 'vfs'
             }
         }),
+        'superuser_service_account_uid': '',
+        'superuser_service_account_public_key': '',
+        '_superuser_service_account_public_key_json': calculate__superuser_service_account_public_key_json,
+        '_superuser_credentials_given': calculate__superuser_credentials_given,
```

Comment:
What is the meaning of the underscores at the start here? Should we document it?